zoomus/components/meeting.py

```diff
@@ -54,7 +54,7 @@ def create(self, **kwargs):
             kwargs['start_time'] = util.date_to_str(kwargs['start_time'])
         return self.post_request(
             "/users/{}/meetings".format(kwargs.get('user_id')),
-            params=kwargs)
+            params=kwargs, data=kwargs.get('data'))
```

Comment:
I presume if this is an error here, then it will likely be an error in all places where we pass in the params as `kwargs`. Also, since you didn't remove 'data' from kwargs, it will be passed twice: once as part of `params` and once as part of `data`. Finally, it looks like this causes the [tests to fail](https://travis-ci.org/actmd/zoomus/builds/591631531?utm_source=github_status&utm_medium=notification).

Reply:
I have tried creating a zoom meeting by sending the "type" and "topic" fields in "params".
   **response = client.meeting.create(start_time=datetime.datetime.now(), user_id=<user_id>,params={'type':2,'topic':'b_meeting'})**. But it didnt worked out. I got an error as follows. **{"code":300,"message":"Request Body should be a valid JSON object."}**.
Then I locally fixed by appending **data=kwargs.get('data')** in create method of meeting.py and sent my request as follows  **response = client.meeting.create(start_time=datetime.datetime.now(), user_id=<user_id>,data={'type':2,'topic':'b_meeting'})**. It worked then.